content/manager/implement-multi-tenancy.md

```diff
@@ -1 +1 @@

####How Cloudify Manager Works with the LDAP/AD Service

When a user logs in to Cloudify Manager, their credentials are passed to the LDAP/AD service for authentication. By default, all users in the LDAP/AD service are authenticated to Cloudify Manager, however only users who have specific permissions for a tenant can access it. When a user logs into Cloudify Manager, the service authenticates the user and returns a list of any groups to which the user belongs. If a Cloudify Manager instance has specified a group name for a tenant that has the same name as a group specified in LDAP/AD, all users in that group can access the tenant (unless an individual user is specifically suspended.) In addition, individual users can be given access to a tenant. The following graphic indicates how Cloudify Manager interacts with an LDAP/AD service. 

![User/LDAP relationship]({{< img "manager/multi-tenancy-ldap-relationship.png" >}})

####Managing Users in the Multi-Tenancy Environment

There are a number of actions related to user access to tenants that an **`administrator`** user can perform. 

{{% gsNote title="Note" %}}
Although Cloudify is for the most-part case-sensitive, user and group names/DNS are case-insensitive, to align with LDAP protocols.
{{% /gsNote %}}

#####Tenant-Related Commands

To run these tenant-specific commands, use `cfy tenant`.

- `create` enables you to create a tenant
- `delete` enables you to delete a tenant
- `get` enables you to view information about the tenant, including its users
- `list` provides a list of all tenants in this instance of Cloudify Manager

#####User-Related Commands

To run these commands, use `cfy users`.

- `add-user` enables you to add an individual user to a tenant
- `add-user-group` enables you to add a group of users to a tenant
- `remove-user` enables you to remove a specific user from a group
- `remove-user-group` enables you to remove a user group from a tenant
- `set-password` enables you to set the password for a user (in the case of a non-LDAP/AD setup)
- `set-role` enables you to set the role for a user. Possible roles are `administrator`, `user`, and `suspended`. The default role assigned to new user is `user`. An `administrator` user can perform all the commands listed here on all tenants in the Cloudfy Manager. Someone with a `user` role has access to all public resources in the tenant(s) to which they are assigned, and to private resources of which they are the owner. A `suspended` user cannot access a tenant.


####Managing Users in Cloudify Manager outside of LDAP/AD

If you choose not to integrate Cloudify Manager with LDAP/AD, you must add each user individually and set a password for them. You can also create groups and add users to groups to them. The users and user groups can be assigned to one or more tenants.






```

Comment:
At the one-before-last sentence - "You can also create groups and add users to groups to them" - we should remove the "to groups".